src/components/v1/scope/services/getScopeById.js

```diff
@@ -0,0 +1,13 @@
+import boom from '@hapi/boom';
+
+import scopeDao from '../DAO';
+
+const getScopeById = async (id) => {
+  const scope = await scopeDao.findScopeById(id);
+
+  if (!scope) throw boom.badRequest(`Scope with ID: ${id} not found`);
```

Comment:
Do you consider to use Enums for the HTTP response?

Reply:
In this case because is JS I do not see necessary @raiandexter0607